src/components/CheckboxList/CheckboxList.jsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import PropTypes from 'prop-types';
+
+import { daysArray } from "../../config/dayNamesForCard";
+
+import {
+    ChooseDayList,
+    ChooseDayItemLabel,
+    ChooseDayItem
+} from "./CheckboxList.style";
+
+const CheckboxList = ({setCheckedDays, checkedDays}) => {
+
+    const changeCheckedDays = (index) => {
+        return setCheckedDays(existingItems =>
+            [
+                ...existingItems.slice(0, index),
+                checkedDays[index] = !checkedDays[index],
+                ...existingItems.slice(index + 1),
+            ]
+        );
+    }
+
+    return <ChooseDayList>
+        {
+            daysArray.map((day, index) =>
+                <ChooseDayItemLabel htmlFor={day} key={day}>
+                    <ChooseDayItem
+                        type='checkbox'
+                        id={index}
+                        value={checkedDays[index]}
+                        onClick={() => changeCheckedDays(index)}
+                        isCheckedBox={checkedDays[index]}
+                    />
+                    {day.charAt(0).toUpperCase() + day.slice(1)}
```

Comment:
it could be done by css `text-transform` or simply make into config `daysArray` in needed way - ["Mon", ....]